core/tracepoint/event_spec.rb

```diff
@@ -0,0 +1,29 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+module ClassSpecs
```

Comment:
It's customary to put these globals in fixtures, and to name them accordingly to avoid any potential conflict with other specs.

Reply:
okay, I have moved the globals in fixtures